tests/migration_test.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pytest
+from unittest.mock import patch
+from migrations.migration import Migration
+from pytest import fixture
+
+
+class mock_response:
+    """Wrap data in response like object"""
+
+    def __init__(self, data, status_code=200):
+        self.data = data
+        self.status_code = status_code
+
+    def json(self):
+        return self.data
+
+    def raise_for_status(self):
+        if self.status_code == 200:
+            return
+        raise Exception("status code ain't 200")
```

Comment:
would be better to mock the exception type and details raised by `requests.raise_for_status`